distribution/linux/flatpak/io.github.openenroth.openenroth.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+DATAFILE="magic7.vid"
+SAVEDIR="$XDG_DATA_HOME/mm7/saves"
+GAMETITLE="Might and Magic VII"
+
+export OPENENROTH_MM6_PATH="$XDG_DATA_HOME/mm6/data"
+export OPENENROTH_MM7_PATH="$XDG_DATA_HOME/mm7/data"
+export OPENENROTH_MM8_PATH="$XDG_DATA_HOME/mm8/data"
```

Comment:
Hmm, is there a way to use externally set `OPENENROTH_MM7_PATH` from inside the flatpak container?

Also, where does the `$XDG_DATA_HOME` point to in this case? Is it a `chroot` fs inside the container, or smth else?

Reply:
There's probably no reason to export anything here, can just pass `--data-path` to `OpenEnroth`?